ADApp/pluginSrc/NDFileHDF5.cpp

```diff
@@ -1658,10 +1658,20 @@ asynStatus NDFileHDF5::closeFile()
               "%s::%s Closing file not totally clean.  Attributes remaining=%d\n",
               driverName, functionName, obj_count);
   }
+  obj_count = (int)H5Fget_obj_count(this->file, H5F_OBJ_ALL);
+  if (obj_count > 0){
+    asynPrint(this->pasynUserSelf, ASYN_TRACE_ERROR,
```

Comment:
We seem to be getting a message `All remaining = 1` here. That might be the file object itself (?) as we only close the file a few lines later. So not sure we should put this as an ASYN_TRACE_ERROR unless the obj_count is greater than 1... Thoughts?

Reply:
The purpose of ASYN_TRACE_ERROR would be that it gets noticed ASAP. Not sure if it does if `asynSetTraceIOMask()` and/or `asynSetTraceMask()` are not called at IOC startup, though.

I can try to change the condition to 1, leaving the rest intact and see what happens.

Reply:
What I mean is that I suspect the number will always be at least 1 - and that's not an error condition because it is the file object itself. Maybe.

Reply:
Understood. We can see if the assumption about the file object holds with the fix at hand. Any newly added _buggy_ code should trip this check, is what I would expect.

Reply:
See https://github.com/areaDetector/ADCore/pull/390#issuecomment-478235586.